packages/ui-react/src/containers/Layout/Layout.tsx

```diff
@@ -38,13 +39,13 @@ const Layout = () => {
   );
   const isLoggedIn = !!useAccountStore(({ user }) => user);
   const favoritesEnabled = !!config.features?.favoritesList;
-  const { menu, assets, siteName, description, styling, features } = config;
+  const { menu, assets, siteName, description, features } = config;
   const metaDescription = description || t('default_description');
 
   const profileController = getModule(ProfileController, false);
 
   const { searchPlaylist } = features || {};
-  const { footerText } = styling || {};
```

Comment:
@langemike i've noticed we removed the styling prop here but left the footerText property on the Styling type. 

1. Can this be removed or do we want to support that feature as well?
2. We should create a ticket to look into styling the footer.

Reply:
The property is removed from the Styling type here: https://github.com/Videodock/ott-web-app/pull/45/files#diff-6dcbc73917653a422864df5ffb7387836f8e4f70ccd73d713f13ac9de51c43bdL62

So, yes we want to remove it.

Styling indeed can be improved, but I think the current state is acceptable. Mostly due the fact since the footer was already implemented, the styling is identical how it previously was. It now only reads it from another source